Celeste.Mod.mm/Mod/Everest/Everest.Updater.cs

```diff
@@ -159,14 +136,34 @@ public class Source {
 
             public static List<Source> Sources = new List<Source>() {
                 new Source {
-                    NameDialog = "updater_src_buildbot",
+                    Name = "updater_src_stable",
+                    Description = "updater_src_release_github",
 
-                    Index = "https://dev.azure.com/EverestAPI/Everest/_apis/build/builds?definitions=3&api-version=5.0",
+                    UpdatePriority = UpdatePriority.High,
 
-                    IsCurrent = () => VersionSuffix.StartsWith("azure-"),
+                    Index = "https://api.github.com/repos/EverestAPI/Everest/releases",
+                    ParseData = GitHubReleasesParser(offset: 700)
+                },
+                new Source {
+                    Name = "updater_src_beta",
+                    Description = "updater_src_release_github",
 
-                    ParseData = AzureDataParser("https://dev.azure.com/EverestAPI/Everest/_apis/build/builds/{0}/artifacts?artifactName=main&api-version=5.0&%24format=zip", 700)
-                }
+                    Index = "https://api.github.com/repos/EverestAPI/Everest/releases",
```

Comment:
Is there a way to avoid triggering the API request cap?
Get this error when I try accessing this url via browser.

```json
{"message":"API rate limit exceeded for 103.138.75.217. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)","documentation_url":"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}
```

Reply:
> For unauthenticated requests, the rate limit allows for up to 60 requests per hour. Unauthenticated requests are associated with the originating IP address, and not the person making requests.

🤔 Maybe we can ask max4805 to fetch releases with access token and cache the results?

Reply:
I think API rate limits is also why Olympus shows a limited amount of commit descriptions in the versions list 🤔 I could definitely fetch and cache the list on my end, the only issue would be that there would be a small delay before a new release appears

Reply:
> There would be a small delay before a new release appears.

We can use webhooks, also the limit is 5000 requests per hour with access tokens so I think we can safely increase fetching frequency.

Reply:
We can either have an API that gives [a preformatted list of versions](https://cdn.discordapp.com/attachments/429775439423209472/1010647544541499512/a.json) that could be used by Everest and Olympus to avoid maintaining the logic twice, or just have an API that returns the releases as-is to keep all logic client-side, what do we prefer? 🤔

Reply:
https://max480-random-stuff.appspot.com/celeste/everest-versions ^^

Reply:
Has been implemented using https://everestapi.github.io/everestupdater.txt to link to the update list source